main.go

```diff
@@ -81,6 +86,69 @@ import (
 	"github.com/networkservicemesh/cmd-nsc/internal/config"
 )
 
+func getResolverAddress() (string, error) {
+	file, err := os.Open("/etc/nsm-dns-config/resolv.conf.restore")
+	if err != nil {
+		return "", err
+	}
+
+	resolverAddr := ""
+
+	scanner := bufio.NewScanner(file)
+	scanner.Split(bufio.ScanLines)
+
+	for scanner.Scan() {
+		cfgLine := strings.Split(scanner.Text(), " ")
+		if cfgLine[0] == "nameserver" {
+			resolverAddr = cfgLine[1]
+			break
+		}
+	}
+
```

Comment:
at this point if `resolverAddr` is still empty string(""), does it makes sense to return error?

Reply:
That should never really happen unless the file is corrupted by something. The container will crash in the grpc connect call anyways if the resolver ip is empty and the resolution fails.